packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -104,15 +104,51 @@ This component renders a list of suggestions coming from the user queries histor
 
 No props are required for the usage of this component.
 
-```vue
-<HistoryQueries />
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work